web/html/src/manager/images/image-view.tsx

```diff
@@ -752,6 +754,20 @@ class ImageViewList extends React.Component<ImageViewListProps, ImageViewListSta
   }
 }
 
+function ImageViewBuildLog(props) {
```

Comment:
Even though this component is tiny, I would recommend extracting out into a separate file similar to `ImageViewOverview` etc. The only reason is that this file is huge as-is and over time it would be good to try to chunk it into better pieces and try and avoid growing it further.